parm/transfer/transfer_gdas_enkf_enkf_05.list

```diff
@@ -24,8 +24,8 @@
 # directory are included, so if no exclude patterns match that file, it will be
 # tranatmferred.
 
-#com/gfs/_ENVIR_/enkfgdas._PDY_/
-/gpfs/dell1/nco/ops/com/gfs/_ENVIR_/enkfgdas._PDY_/
+#_COMROOT_/gfs/_SHORTVER_/enkfgdas._PDY_/
```

Comment:
I know these are ops-only, but can we delete all these commented-out lines from these .list files? This one-liner should do it (but do it in a clean repo so you can reset if not):
`perl -i -p -e "s/^#_.*$//g *.list`